renovate-config.js

```diff
@@ -24,11 +24,11 @@ module.exports = {
     onboarding: false,
     requireConfig: false,
     constraints: {
-        pnpm: "< 7.0.0",
+        pnpm: "< 8.0.0",
     },
     allowedPostUpgradeCommands: [".*"],
     postUpgradeTasks: {
-        commands: ["npm i -g pnpm@'< 7.0.0' && pnpm install && pnpm build"],
```

Comment:
I've opted for a specific version (like the github workflows) if we do that, do we need the constraint?

Reply:
@geofflamrock

Reply:
Hmmm not sure, I _think_ we might be just able to specify the constraint as a specific version and it will use that? I'm not sure why we would have done it differently when we set it last time though, so perhaps that doesn't really work. Might need to do some investigation in the renovate docs to see how it can be configured.

Reply:
Ah I remember why we have this extra thing here now. github actions requires us to have a built javascript version of the code committed to the repo in order for it to work (yep I know, annoying), so when renovate runs we need it to produce built javascript to include in the PR, so we use this post upgrade task to achieve that. Makes sense that we want a specific version of pnpm for that so that we have consistency, but it isn't necessarily connected to the constraint, that is used for other renovate actions. Still worth seeing whether we can pin that constraint to the same version for consistency I expect.

Reply:
Ah ok that makes sense, the docs don't really specify anything specific when talking about the "postUpgradeTasks" because of course they can be anything. The docs say that you can put what ever version you want in the constraint so I'll make both of these values align with the github works flows 👍